nservicebus/azure-storage-persistence/configuration_sections_azure_[,6.2].partial.md

```diff
@@ -0,0 +1,5 @@
+### Configuration with Configuration Section
```

Comment:
@Scooletz I'm a bit confused: is configuration_sections_azure_[,6.2].partial.md and configuration_sections_asp_[1,].partial.md the same partial? Similar for code partial? 

Can you clarify how it should be rendered, I have a feeling we can simplify that a bit, but not sure if I got your intention right?